src/mount/sauna_client.h

```diff
@@ -83,8 +83,10 @@ struct FsInitParams {
 	static constexpr bool     kDefaultIgnoreFlush = false;
 #ifdef _WIN32
 	static constexpr unsigned kDefaultWriteCacheSize = 50;
+	static constexpr unsigned kDefaultWriteWaveTo = 10;
 #else
 	static constexpr unsigned kDefaultWriteCacheSize = 0;
+	static constexpr unsigned kDefaultWriteWaveTo = 50;
```

Comment:
Off topic: Why the differences between Windows and Linux?
Windows: 50 MiB of write cache by default and smaller wave timeout.
Linux: Write cache disabled by default and bigger wave timeout.

Reply:
It is a very interesting question, glad you asked:
- the Linux default cache size though it says it is "disabled", it is not. The main function of the Linux client shows this:
```cpp
	if (!gMountOptions.writecachesize)
		gMountOptions.writecachesize = 128;
```
so, guess what: the default Linux write cache size is actually 128MiB (the client write algorithm does not work if no cache provided). This was something I found out several months ago and found it very misleading. I changed the default value to the real 128.
- the Windows default cache size of 50MiB comes from 7 years ago implementation, and I didn't touch it. Probably it was lower because they were testing it in some VM? I don't know. I change it to 128, so now both default values are equal and it could be taken off the preprocessor directives.
- the differences in the write wave timeout came from the following: the Windows is not able (well, in fact the thing is that it is very expensive) to notify the write executors that a new cached block must be written. This new cached block could be pushed into the datachain the ChunkJobWriter is writing from the write_data - write_blocks - write_block sequence of functions while the ChunkJobWriter is waiting for the operations to finish. 

So, the idea is kind of: the Windows can not know if a new block is already cached and ready to be sent to the chunkservers, so it checks much more frequently, on the other hand, the Linux can know it, and it is relieved that it will be notified, so this timeout will only have to take into consideration that the operations are not able to be done in this time and sets the value not so high to not waste much time if there is some error from the chunkserver side.